### DISCUSSION TITLE
Public development server 🖥

### DISCUSSION THREAD

**Discussion by Vdauphin** [Vdauphin/HeartsAndMinds#1098] [Show and tell]
### 📣 Presentation:
It's been a while I opened a public development server to test feature in a real world situation.
You guys are free to team up and play on it!

### 📯 Disclaimers:
- This server autosave on player disconnect so everything you do, good or bad, will persist.
- As this server is for testing, expect lag, bug and service interruption.

### 📄 Informations:
**Server name:** Hearts and Minds [ACE/ACRE]
**Map:** Tanoa
**Address:** [heartsandminds.ddns.net:2302](https://www.battlemetrics.com/servers/arma3/8760384)
[**Mods:**](https://github.com/Vdauphin/HeartsAndMinds/files/7835692/Hearts_and_Minds.txt) CBA, ACE and ACRE

<details>
<summary>Pictures of map progress</summary>
<p>

![20230204115608_1](https://user-images.githubusercontent.com/14364400/216763493-266e73bb-ddec-4b30-b685-de2900bbfe77.jpg)

</p>
</details>

**Comment by JWB502** [comment]
YES! ACRE! I'll save it to my favorites.

**Comment by Raz0rLegend** [comment]
Nice ! I'll come soon i can !

**Comment by STyx2909** [comment]
Still available?

**Reply by Vdauphin** [reply]
Yes
